### PR TITLE
added GF dual port SRAM tech map

### PR DESCRIPTION
**Pull request by jake-ke**
This only adds the dual-port techmap which defaults its switch to False.

### CODE REVIEW

File: lake/top/tech_maps.py [hunk 1]
```diff
@@ -108,13 +111,62 @@ def GF_Tech_Map(depth, width) -> dict:
...
+        }
+    else:
+        ports.append(single_port)
+
+        tech_map = {
+            'name': f"IN12LP_S1DB_W{depth:05}B{width:03}M04S2_H",
+            'ports': ports,
+            'depth': depth,
+            'width': width
+        }
```
Comment (mbstrange2) [inline lake/top/tech_maps.py:170]: there's some redundant code here - can set name and add the ports in the if/else then set the final tech map after

**Review by mbstrange2 — CHANGES_REQUESTED** [review]
Address the changes and I'll approve

File: lake/top/tech_maps.py (hunk 1 above)
Comment (jake-ke) [reply lake/top/tech_maps.py]: Thanks for the review!

**Review by mbstrange2 — APPROVED** [review]
LGTM

**Pull request merged**